src/components/AvatarCropModal/gestureHandlerPropTypes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    current: PropTypes.shape({
+        eventNames: PropTypes.arrayOf(PropTypes.string),
+        reattachNeeded: PropTypes.bool,
+        registrations: PropTypes.arrayOf(PropTypes.number),
+        viewTag: PropTypes.number,
+        worklet: PropTypes.func,
```

Comment:
It is a propType of a react-native-gesture-handler. I've created it based on a structure of an object that a gesture handler returns because this lib does not have its propTypes. We are not using these props in our code. Also, I don't know what they are doing, and there is no documentation for them. 
@parasharrajat @tgolen Is it possible to leave these props uncommented? If yes, what comment should I add?

Reply:
In that case, I would expect this to use Proptype.object. Let's wait for @tgolen's answer.